ECMA2Yaml/ECMAHelper/YamlConverters/SDPYamlConverter.Overload.cs

```diff
@@ -48,13 +48,27 @@ public OverloadSDPModel FormatOverload(Member overload, List<Member> members)
             sdpOverload.DevLangs = sdpOverload.Members.SelectMany(m => m.DevLangs).Distinct().ToList();
             sdpOverload.Monikers = sdpOverload.Members.Where(m => m.Monikers != null).SelectMany(m => m.Monikers).Distinct().ToList();
 
+            bool resetMemberThreadSafety = false;
```

Comment:
I think here the logic should be, if the overload itself has ThreadSafetyInfo, we should show only that, and hide all thread safety in its members. If not, we should continue to show each member's own value.